tlog-checkpoint.md

```diff
@@ -0,0 +1,74 @@
+# Transparency Log Checkpoints
+
+A checkpoint is a [signed note][] where the body is precisely formatted for use
+in transparency log applications.  The mandatory note text includes the three
+essential parts of a log's Merkle tree head at a given size.
+
+```
+example.com/behind-the-sofa
```

Comment:
FWIW: I think this gives unnecessary CT vibes. I'd rather have a more neutral example, e.g., "example.org/some-log" or similar. Like you did with the example witness in cosignature.md. If you update here, please also update the log's example origin line in cosignature.md.

Reply:
Only a few folks will catch the reference :) and even then, I think amalgamating CT into the checkpoint ecosystem is a good thing overall.